test/src/cookies.spec.ts

```diff
@@ -750,5 +778,87 @@ describe('Cookie specs', () => {
       // Expect the cookie for the COOKIE_DESTINATION from URL_1 is deleted.
       expect(await page.cookies(COOKIE_DESTINATION_URL)).toHaveLength(0);
     });
+    it('should only delete cookie without partition key if partitionkey is not specified', async () => {
+      const {page, server} = await getTestState();
+      const url = 'https://example.com';
+
+      await page.goto(server.EMPTY_PAGE);
+      await page.setCookie({
+        url,
+        name: 'partitionCookie',
+        value: 'partition',
+        partitionKey: 'http://localhost',
+      });
+
+      await page.setCookie({
+        url,
+        name: 'partitionCookie',
+        value: 'partition',
+      });
+
+      await page.deleteCookie({
+        url,
+        name: 'partitionCookie',
+      });
+
+      expect(await page.cookies(url)).toHaveLength(1);
+      await expectCookieEquals(await page.cookies(url), [
+        {
+          name: 'partitionCookie',
+          value: 'partition',
+          domain: 'example.com',
+          path: '/',
+          expires: -1,
+          size: 24,
+          httpOnly: false,
+          secure: true,
+          session: true,
+          sameParty: false,
+          sourceScheme: 'Secure',
+          partitionKey: 'http://localhost',
+        },
+      ]);
+    });
+    it('should only delete cookie with partition key if partition key is specified', async () => {
```

Comment:
I think same here, the partition key cannot be localhost since the page is at localhost (default partition key).

Reply:
done. changed to https://example.com/.